### PR TITLE
Add some missing changes to the ChangeLog

### PR DESCRIPTION
**Pull request by softins**

### CODE REVIEW

**Review by hoffie — APPROVED** [review]
Looks good.

Should we have "coded by" entries? Feel free to merge anyway, I plan to send another PR for Changelog adjustments anyway. :)

File: ChangeLog [hunk 1]
```diff
@@ -55,6 +55,16 @@
...
   Ignore the server version if it is not a release, due to a suffix such as dev, alpha or beta.
 
+- Windows installer interface now supports English, Dutch, French, German, Polish and Spanish.
+
+- bug fix: Fixed some memory leaks in server recording (#1073)
+
+- bug fix: Adjusted some accelerator keys in translations to avoid duplication (#1165)
+
+- Release now builds .deb installation packages for Ubuntu/Debian (#1100)
+  Compatible with Ubuntu 18 and 20.
```
Comment (hoffie) [inline ChangeLog:65]:
```suggestion
  Compatible with Ubuntu 18.04 and 20.04.
```

File: ChangeLog [hunk 2]
```diff
@@ -55,6 +55,16 @@
...
+
+- Release now builds .apk installation packages for Android (#880)
```
Comment (hoffie) [inline ChangeLog:67]:
```suggestion
- Release now builds experimental .apk installation packages for Android (#880)
```

**Review by hoffie — CHANGES_REQUESTED** [review]
Sorry, was a bit quick. Just saw two things which I think could be adapted. :)

**Comment by softins** [comment]
> Looks good.
> 
> Should we have "coded by" entries? Feel free to merge anyway, I plan to send another PR for Changelog adjustments anyway. :)

We should certainly include "coded by" entries for new features, but probably not for bug fixes.

File: ChangeLog (hunk 2 above)
Comment (softins) [reply ChangeLog]: Done

File: ChangeLog (hunk 1 above)
Comment (softins) [reply ChangeLog]: Done

**Review by hoffie — APPROVED** [review]

**Review by hoffie — APPROVED** [review]

**Review by ann0see — CHANGES_REQUESTED** [review]
The flatpak build has been removed since we're on flathub now. So you could remove it

**Review by ann0see — APPROVED** [review]
Yes, I think we can leave it like that.

**Pull request merged**